css/style.css

```diff
@@ -0,0 +1,344 @@
+/* css reset */
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  -ms-text-size-adjust: 100%;
+  -moz-text-size-adjust: 100%;
+  -webkit-text-size-adjust: 100%;
+  vertical-align: baseline;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-image: url(../images-apple/back.png);
+  background-repeat: no-repeat;
+  min-width: 1349px;
+  min-height: 567px;
+  cursor: pointer;
+}
+
+main {
+  margin-top: 115px;
+}
+
+.none {
+  display: none;
+}
+
+p {
+  padding-left: 0;
+  color: white;
+  font-size: 12px;
+  font-family: "Lucida Grande", "Lucida Sans Unicode", Verdana, sans-serif;
+  margin-top: 10px;
+  font-weight: 100;
+}
+
+a {
+  text-decoration: none;
+}
+
+.nav-container {
+  display: flex;
+  border-radius: 10px;
+  position: relative;
+  padding-left: 0;
+  align-items: center;
+  justify-content: center;
+}
+
+nav div {
+  display: flex;
+  align-items: center;
+  min-width: 106px;
+  max-height: 36px;
```

Comment:
please change this to height: 40px; it will fix it form this:
![image](https://user-images.githubusercontent.com/34653764/85241825-0813c080-b435-11ea-8d95-a2ab8b064fe5.png)
to this:
![image](https://user-images.githubusercontent.com/34653764/85241808-f9c5a480-b434-11ea-85ea-ce2fb29439d0.png)